tests/crawler/test_crawler_unittest.py

```diff
@@ -28,3 +32,38 @@ def test_close_all_windows_except_first(self):
         crawler.close_all_windows_except_first(self.driver)
 
         assert (len(self.driver.window_handles) == 1), 'some windows were not closed properly.'
+
+
+class TestCrawlerLive(unittest.TestCase):
+    @classmethod
+    def setUpClass(cls):
+        cls.server = multiprocessing.Process(target=run_server)
+        cls.server.start()
+        sleep(1)
```

Comment:
i think the proper solution would actually be using selenium, as we do so well: [selenium wait](https://selenium-python.readthedocs.io/waits.html).
however, since we already rely on `sleep` in the source itself, it thought this would be good enough for now. for example:
https://github.com/mozilla/coverage-crawler/blob/bdf705716c59af34a46dca8d31824839fd932da7/coverage_crawler/crawler.py#L51

it might be better to refactor all `sleep` into `EC.*` in a separate PR. what do you think?

Reply:
selenium wait wouldn't be usable when the server hasn't even started yet. The example script on that link would fail at `get`.

Reply:
Unfortunately that sleep is kind of required as our crawler is dealing with random pages from the web, so it can't know what to wait for.

There is another sleep later at line 140, added as we couldn't find another way to know when the scroll into view was done.

Better to avoid introducing more sleeps in the codebase if we can avoid it.